lawnchair/src/app/lawnchair/search/LawnchairAppSearchAlgorithm.kt

```diff
@@ -149,6 +162,9 @@ class LawnchairAppSearchAlgorithm(context: Context) : LawnchairSearchAlgorithm(c
         return createSearchTarget(id, action, extras)
     }
 
+    private fun Sequence<AppInfo>.showHiddenApps(): Sequence<AppInfo> =
+        filter { showHiddenAppsInSearch || it.toComponentKey().toString() !in hiddenApps }
```

Comment:
How about having an if condition outside of the `filter` to make it easier to read? Something like this:

```kotlin
if (showHiddenAppsInSearch) return this
return filter { /* filter logic */ }
```

Reply:
94f690e85261cb7457f417cf876dcde802899cc5